### DISCUSSION TITLE
String

### DISCUSSION THREAD

**Discussion by clente** [clente/pt-bR#2] [Glossary]
Qual é a melhor tradução para "_character string_"? A tradução acadêmica seria "cadeia de caracteres", mas eu vejo pouquíssimas pessoas usando esse termo; na prática eu vejo muito mais "string" ou "string de caracteres".

A tradução atual está inconsistente e existem exemplos das 3 opções.

**Comment by rmhirota** [comment]
Um ano depois, eu decidi que talvez _sequência de caracteres_ faça sentido tb.
